custom/icds_reports/static/js/directives/download/download.directive.js

```diff
@@ -311,6 +311,8 @@ function DownloadController($rootScope, $location, locationHierarchy, locationsS
     };
 
     vm.submitISSNIPForm = function(csrf_token) {
+        $rootScope.issnip_report_link = '';
```

Comment:
Why does this need to be reset?

Reply:
@emord When I tested PDF report I noticed that when you try generate next report then we have two messages.